apps/web/scripts/docker/read-secrets.sh

```
@@ -2,15 +2,44 @@
 
 set -eu
 if [ -f "/run/secrets/database_url" ]; then
-  export DATABASE_URL=$(cat /run/secrets/database_url)
+  IFS= read -r DATABASE_URL < /run/secrets/database_url || true
+  DATABASE_URL=${DATABASE_URL%$'\n'}
```

Shellcheck (0.10.0)

[warning] In POSIX sh, $'..' is undefined.

(SC3003)